app/frontend/components/common/nufiScreenLink.tsx

```diff
@@ -0,0 +1,11 @@
+import {h} from 'preact'
+
+const NufiScreenLink = () => {
```

Comment:
naming nitpick - "page" sounds more natural to me as it is de-facto a separate page  from the rest
```suggestion
const NufiPageLink = () => {
```